e2e/docker_test.go

```diff
@@ -0,0 +1,110 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"syscall"
+	"testing"
+
+	"github.com/ory/dockertest/v3"
+	"github.com/ory/dockertest/v3/docker"
+)
+
+/* -------------------- Dockertest Ephemeral PATH Container Setup -------------------- */
+
+const (
+	containerName        = "path"
+	internalPathPort     = "3000"
+	dockerfilePath       = "../Dockerfile"
+	configFilePath       = "./.config.test.yaml"
+	configMountPoint     = ":/app/.config.yaml"
+	containerEnvImageTag = "IMAGE_TAG=test"
+	containerExtraHost   = "host.docker.internal:host-gateway"
+	timeoutSeconds       = 120
+)
+
+var (
+	containerConfigMount     = filepath.Join(os.Getenv("PWD"), configFilePath) + configMountPoint
+	containerPortAndProtocol = internalPathPort + "/tcp"
+)
+
+func setupPathDocker() (*dockertest.Pool, *dockertest.Resource, string) {
```

Comment:
Please add godoc on how much this does (for future devs)

Reply:
Done.